library/Fission/User/CID/Query.hs

```diff
@@ -0,0 +1,24 @@
+module Fission.User.CID.Query
+  ( getUserCidsByUserId
+  , getUserCidsByCids
+  ) where
+
+import           Network.IPFS.CID.Types
+import           Database.Esqueleto hiding ((=.), update)
+
+import           Fission.Models
+import           Fission.Prelude
+
+-- | CIDs associated with a user
+getUserCidsByUserId :: MonadDB m => UserId -> Transaction m [Entity UserCid]
+getUserCidsByUserId userId =
+  select <| from \userCid -> do
```

Comment:
While we're in here, maybe switch this to `selectFirst`. This will also change the return type to `Transaction m (Maybe (Entity UserCid))`